frontend/index.js

```diff
@@ -0,0 +1,67 @@
+document.addEventListener("DOMContentLoaded", ()=>{
+    let ClassDiv = document.querySelector("#newClassDiv")
+    let ClassForm = document.querySelector("#newClassForm")
+    let StudentDiv = document.querySelector("#newStudentDiv")
+    let StudentForm = document.querySelector("#newStudentForm")
+    let SortDiv = document.querySelector("#sortDiv")
+    let SortForm = document.querySelector("#sortForm")
+    let Show = document.querySelector("#show")
+    let addClassDiv = document.querySelector("#addClassDiv")
+    let addStudentDiv = document.querySelector("#addStudentDiv")
+    console.log("pie");
+
+    ClassForm.addEventListener("submit", async (e)=>{
+        e.preventDefault();
+        addClassDiv.innerHTML = ""
+        let theClass = document.querySelector("#newClass").value
+        let theTeacher = document.querySelector("#newTeacher").value
+        let p = document.createElement("p")
+        let res = await axios.post(`http://localhost:3000/class`,{teacher: theTeacher, name: theClass})
+        p.innerText = res.data.message
```

Comment:
after you make your axios request you can set `theClass` and `theTeacher` to an empty string. That will clear the form and it's more user-friendly that way.

Reply:
Oh man, Thanks . I spent a while trying to figure out a way